basin3d/synthesis.py

```diff
@@ -618,7 +604,7 @@ def get_timeseries_data(synthesizer: DataSynthesizer, location_lat_long: bool =
                 )
 
             if not results:
-                logger.info(f'{synthesized_variable_name} returned no data.')
+                logger.debug(f'{synthesized_variable_name} returned no data.')
```

Comment:
What is the rationale for changing to debug? seems like users might want to know this. Perhaps revisit when we reconsider how to handle cases with no data.

Reply:
I can't remember why I did this but you are right. Can you make a suggestions in the github ui that I may incorpriate